simplivity/resources/omnistack_clusters.py

```diff
@@ -130,9 +130,27 @@ def get_connected_clusters(self):
         connected_clusters = self._client.do_get(method_url).get("omnistack_clusters", [])
 
         clusters = [self._clusters.get_by_id(cluster["id"]) for cluster in connected_clusters]
-
         return clusters
 
+    def get_throughput(self, destination_id=None, time_offset=0, range=43200):
+        """Calculates the throughput between each pair of omnistack_clusters in the federation
+
+Args:
```

Comment:
The alignment of Args is incorrect, please correct the alignment.

Reply:
Updated the alignment correctly